Automattic/sensei
# Fix menu for Settings tabs

The problem appeared with these [changes](https://github.com/Automattic/sensei/commit/f78f6a5e7041b4258bf85c640575a06dfa653adc).
Currently, it doesn't show selected tab.

Didn't manage to fix jumping to the anchor when we click on tab with a URL hash.

### Changes proposed in this Pull Request

* Highlight current tab on Sensei settings page.

### Testing instructions

* Go to Sensei LMS -> Settings.
* Click on Courses tab.
* Make sure it is highlighted (the tab name is bold, has "current" class).
* Click on Emails.
* Make sure it is highlighted when page loaded.


### Screenshot / Video
<img width="719" alt="CleanShot 2023-07-02 at 16 53 41@2x" src="https://github.com/Automattic/sensei/assets/329356/2e393bc3-a8cc-4feb-b6ed-4e8c9f31aab6">
